libs/transactions/src/lib/components/transaction-item/transaction-item.component.ts

```diff
@@ -46,3 +72,8 @@ export class TransactionItemComponent implements OnChanges {
     }
   }
 }
+
+@Directive({
+  selector: '[bbTransactionsItemAdditions]'
+})
+export class TransactionItemAdditonalDetailsDirective extends ExtensionSlotDirective<TransactionItem> {}
```

Comment:
I assume this type should be TransactionAdditionalDetailsContext now, instead of TransactionItem, and exampleChanges$ should also not return the entire transaction item